js/app/dashboard/redux/middlewares.js

```diff
@@ -104,5 +105,10 @@ export const persistFilters = ({ getState }) => (next) => (action) => {
     window.sessionStorage.removeItem(`cpccl__dshbd__fltrs__${getKey(state)}`)
   }
 
+ if (action.type === SHOW_RECURRENCE_RULES) {
+    state = getState()
+    window.sessionStorage.setItem(`recurrence_rules${getKey(state)}`, JSON.stringify(state.settings.isRecurrenceRulesVisible))
```

Comment:
The key should be just `recurrence_rules_visible`, as it's not linked to the date